res/values/strings.xml

```diff
@@ -510,4 +510,21 @@
 <string name="sms_send">Send</string>
 <string name="sms_send_failure">SIP SMS sending failed</string>
 
+    <string name="silent_config">Configure Silent Mode</string>
+    <string name="silent_disable_enable">Disable/Enable</string>
+    <string name="silent_on">Silent Mode On</string>
+    <string name="silent_summary_off">Silent Mode is Off"</string>

```

Comment:
Consistency here too; either label "silent_summary_on" and "silent_summary_off" or both "silent_on" and "silent_off" (I always prefer shorter)